rules/prefer-object-define-properties.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+const isSameReference = require('./utils/is-same-reference.js');
+const {removeArgument} = require('./fix/index.js');
+
+const MESSAGE_ID = 'prefer-object-define-properties';
+const messages = {
+	[MESSAGE_ID]: 'Prefer `{{replacement}}` over multiple `{{value}}`.',
+};
+
+function getFirstExpression(node, sourceCode) {
+	const {parent} = node;
+	const visitorKeys = sourceCode.visitorKeys[parent.type] || Object.keys(parent);
+
+	for (const property of visitorKeys) {
+		const value = parent[property];
+		if (Array.isArray(value)) {
+			const index = value.indexOf(node);
+
+			if (index !== -1) {
+				return value[index - 1];
+			}
+		}
+	}
+}
+
+function isObjectDefineProperty(node) {
+	return (
+		node
+		&& node.type === 'CallExpression'
+		&& node.callee.type === 'MemberExpression'
```

Comment:
```suggestion
		&& node.callee.type === 'MemberExpression'
		&& !node.computed
````

`Object[defineProperty](...)`

Reply:
Please add test for it.

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/pull/1843/files#diff-9303f48cf071a86d51ccd26a329a6dcb887acc1bb20a64298170dcd3e2c5d2a0R78-R83

I can only think of something like tihs